docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Azure
+description: Instructions to infer an ONNX model remotely with an Azure endpoint
+parent: Execution Providers
+nav_order: 12
+redirect_from: /docs/reference/execution-providers/Azure-ExecutionProvider
+---
+
+# Azure Execution Provider (Preview)
+
+The Azure Execution Provider enables ONNX Runtime to visit an remote endpoint for inferenece, the endpoint must be deployed beforehand.
+Azure Execution Provider is in preview stage, all API(s) and usage are subjuct to change.
+
+One use case for Azure Execution Provider is small-big models. E.g. A smaller model deployed on edge device for faster inference,
+while a bigger model deployed on Azure for higher precision, with Azure Execution Provider, a switch between the two could be easily achieved.
+Note that the two models are expected to have same inputs and outputs.
+
+## Limitations
+
+To consume the endpoint, ONNX Runtime must load the model locally, it could be the same model deployed to Azure, or another model that has exactly same inputs and outputs.
```

Comment:
The intro para here does not feel like a limitation (and makes the EP look more limited than it is). Can we move the para outside of the limitations section?

Reply:
Moved to description.